Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.cpp

```diff
@@ -71,80 +79,82 @@ Cursor::ReshapeData Cursor::GetReshapableState(bool adjusted)
   const ControlState y = controls[0]->GetState() - controls[1]->GetState();
   const ControlState x = controls[3]->GetState() - controls[2]->GetState();
 
-  // Return raw values. (used in UI)
+  // Return raw values
   if (!adjusted)
     return {x, y};
 
-  return Reshape(x, y, 0.0);
+  return Reshape(x, y, 0.0, std::numeric_limits<ControlState>::infinity());
```

Comment:
Because with a relative axis, like a mouse, the offset from the previous frame can, and should, be up to any number. It made no sense that the maximum speed between two frames was 1.

Reply:
You're right. This value is processed further by `GetState` and should not be clamped here.
I think you should add a comment explaining this though.

Reply:
Should I? It made no sense to clamp it in the first place 😅.

Reply:
Looking at this again, seems like it's now also not clamped when NOT using relative input. I think I had thought of that already but I need to test again.

Reply:
That should be fine as long as the final `GetState` function clamps I think.

Reply:
It does behave differently without the clamp, it can go over the borders in wiimote Widget and in the game I guess, Not sure what the consequences would be. I don't even understand why it's reshaped when using the relative movement setting.

Reply:
`Reshape` also processes the calibration so it should still be used with relative input.
And on second though it should be clamped in the end so relative input doesn't let the pointer just float off the screen forever.

Reply:
The relative one is already clamped in GetState(). The non relative one isn't clamped anymore, not as before, but it might still be "clamped" by other means in the Reshapre function. Will confirm tomorrow as I feel like there is a reason I have changed it for both relative and non relative settings.

Reply:
All right, I think it should still be clamped when not in relative mode, otherwise you could go over the max specified angle, which usually corresponds to the borders of the game view and the borders of your window, if you want to go over these values, it's better to just increase the max angles instead of accepting values over abs(1).
And it should always be clamped when in relative mode to avoid the cursor drifting over the game view.
Will fix later.

Reply:
Fixed